src/Components/MoreHelp/MoreHelp.tsx

```diff
@@ -1,45 +1,43 @@
 import { FormattedMessage } from 'react-intl';
+import { useConfig } from '../Config/configHook.tsx';
 import './MoreHelp.css';
 
 type Resource = {
-  name: string;
-  description?: string;
+  name: JSX.Element;
+  description?: JSX.Element;
   link?: string;
-  phone?: string;
+  phone?: JSX.Element;
+  label?: string;
 };
 
 const MoreHelp = () => {
-  const resources: Resource[] = [
-    {
-      name: '2-1-1 Colorado',
-      link: 'https://www.211colorado.org',
-      phone: 'Dial 2-1-1 or 866.760.6489',
-    },
-    {
-      name: 'Family Resource Center Association',
-      description:
-        'Your local family resource center may be able to connect you to other resources and support services. Visit a center near you.',
-      link: 'https://maps.cofamilycenters.org',
-    },
-  ];
+  const { moreHelpOptions } = useConfig('more_help_options');
+  const resources: Resource[] = Object.values(moreHelpOptions);
 
   const displayResources = (resources: Resource[]) => {
-    return resources.map((resource) => {
+    return resources.map((resource, index) => {
       return (
-        <article key={resource.name} className="resource-card-article">
-          <h1 className="resource-header" key={resource.name}>
-            {resource.name}
+        <article key={index} className="resource-card-article">
+          <h1 className="resource-header" key={index}>
+            <FormattedMessage id={resource.name.props.id} defaultMessage={resource.name.props.defaultMessage} />
```

Comment:
```suggestion
            {resource.name}
```
The config transforms it into a formatted message, so you can just inline it.